README.md

```diff
@@ -62,6 +62,52 @@ Contacts.getContact( contactIdentifier, (error, contact) =>  {
 });
 ```
 
+#### Delete a Single Contact
+
+```js
+let contactIdentifier = 'A7806266-6574-4731-82E1-C54946F63E1C';
+
+Contacts.deleteContact( contactIndetifier, (error, contactId) => {
+  if (error) {
+    console.log(error);
+  }
+  else {
+    console.log(contactId);
+  }
+}
+```
+
+#### Create a Single Contact
+
+```js
+let data = {
+  'givenName': 'John',
+  'familyName': 'Appleseed',
+  'organizationName': 'Apple Inc',
+  'phoneNumbers': [
+    // (NOTE)
+    // Available labels 'home', 'work', 'mobile'
```

Comment:
Instead of using a string for label and letting people know what the possible values are, can you export those labels as Constants. Take a look at this: https://facebook.github.io/react-native/docs/native-modules-ios.html#exporting-constants